chunky3d/chunky.py

```diff
@@ -495,12 +497,14 @@ def update_grid_mask(self):
             type(self._grid_mask) is not np.ndarray
             or self._grid_mask.shape != self._block_shape
         ):
-            self._grid_mask = np.full(self._block_shape, 0xFFFFFFFF, dtype=np.uint32)
+            self._grid_mask = np.full(
+                self._block_shape, Sparse.EMPTY_GRID_VALUE, dtype=np.uint32
+            )
         else:
-            self._grid_mask.fill(0xFFFFFFFF)
+            self._grid_mask.fill(Sparse.EMPTY_GRID_VALUE)
 
-        self._grid_mask[tuple(np.array(sorted(self._grid.keys())).T)] = np.arange(
-            self.nchunks_initialized
+        self._grid_mask[tuple(np.array(sorted(self._grid.keys())).T)] = (
+            np.arange(self.nchunks_initialized)
```

Comment:
Is it `black`?

Reply:
It was, I just had different line length settings. Reverted the change.